models/pull.go

```diff
@@ -26,8 +26,8 @@ var PullRequestQueue = sync.NewUniqueQueue(setting.Repository.PullRequestQueueLe
 type PullRequestType int
 
 const (
-	PULL_REQUEST_GOGS PullRequestType = iota
-	PLLL_ERQUEST_GIT
+	PULL_REQUEST_GITEA PullRequestType = iota
```

Comment:
Can we use a separate PR for cleaning that up ? I'm just focusing on ^GOGS^GITEA here...

(and maybe we could use some automatici linting check to spot those)